Examples/SAM/SQSLambda/Lambda.swift

```diff
@@ -0,0 +1,40 @@
+// ===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// ===----------------------------------------------------------------------===//
+
+import AWSLambdaEvents
+import AWSLambdaRuntime
+import Foundation
+
+@main
+struct SQSLambda: SimpleLambdaHandler {
```

Comment:
when utilizing the `init(context:` you should switch from `SimpleLambdaHandler` to `LambdaHandler`, that is the main difference between them

Reply:
done